### PR TITLE
DietPi-Software | CUPS

### PR DESCRIPTION
**Pull request by Joulinar**
DietPi-Software | CUPS - add new software title

**PR:** https://github.com/MichaIng/DietPi/pull/4255

### CODE REVIEW

**Comment by StephanStS** [comment]
@Joulinar: There should also be a section within software/printing.md. Do you plan to add this?  
If you want to, I can assist with the editing, we could do this together via Telecon.

If you are working on the PR you could set the PR to "draft" to show that the PR is not finished yet.
grafik

**Comment by Joulinar** [comment]
I know @StephanStS 😉 

It was planned to finish but I got interrupted by my kids destroying my house.

**Comment by Joulinar** [comment]
@StephanStS 
@MichaIng 
I can create a screen print from CUPS GUI. But I'm not sure how to upload 🙄

**Comment by Joulinar** [comment]
maybe I found it. Let me try

**Comment by StephanStS** [comment]
You can send it to me via email, I then add it via PR to your branch.

-----Original Message-----

File: docs/software/printing.md [hunk 1]
```diff
@@ -60,4 +61,24 @@ Website: <https://octoprint.org>
 Official documentation: <https://docs.octoprint.org>  
 Forum: <https://community.octoprint.org>
 
+## CUPS
+
+CUPS provides a web interface for managing your local or network printers.
+
+![CUPS web interface screenshot](../assets/images/dietpi-software-printserver-cups.png){: width="500" height="385" loading="lazy"}
+
+=== "Access the web interface"
```
Comment (StephanStS) [inline docs/software/printing.md:70]: Harmonization: In many other documents we call it "Access to the web interface". Would you please also change this in the octoprint section?

File: docs/software/printing.md [hunk 2]
```diff
@@ -60,4 +61,24 @@ Website: <https://octoprint.org>
...
+
+    The web interface is accessible via port **631**:
+
+    - URL = `http(s)://<your.IP>:631`  
+    - Username = `root`
+    - Password = `<RootPassword>`
```
Comment (StephanStS) [inline docs/software/printing.md:76]: Harmonization: `<your global password>`

**Review by StephanStS — CHANGES_REQUESTED** [review]

File: docs/software/printing.md (hunk 2 above)
Comment (Joulinar) [reply docs/software/printing.md:76]: It's indeed the root password and not the global application pw

File: docs/software/printing.md (hunk 1 above)
Comment (Joulinar) [reply docs/software/printing.md:70]: Yes

**Review by StephanStS — APPROVED** [review]

**Pull request merged**